lib/Monads/Fun_Pred_Syntax.thy

```diff
@@ -173,6 +173,18 @@ lemmas pred_neg_bot_eq[simp] =
    entirely in the future *)
 
 
+subsection "Simplification Rules for Lifted And/Or"
+
+lemma bipred_disj_op_eq[simp]:
+  "reflp R \<Longrightarrow> ((=) or R) = R"
```

Comment:
Nice. If we add this one, we should probably add the commuted version as well to the simp set (`R or (=)`), so that it rewrites reliably.

(edit: just saw that this lemma is only moved; would still be nice to add the other one for completeness)